cli/pcluster/pcluster.py

```diff
@@ -168,6 +162,20 @@ def create(args):
             utils.delete_s3_bucket(bucket_name=batch_temporary_bucket, aws_client_config=aws_client_config)
         sys.exit(1)
 
+
+def _print_stack_outputs(stack):
+    outputs = stack.get('Outputs', [])
+    parameters = stack.get('Parameters')
+    ganglia_enabled = is_ganglia_enabled(parameters)
+    for output in outputs:
+        output_key = output.get('OutputKey')
+        if not ganglia_enabled and output_key.startswith('Ganglia'):
```

Comment:
Instead of growing the number of ifs I would have a static list with the Outputs we want to whitelist and print it if the key belongs to this list. (for ganglia we can add the key to the list after we check if ganglia is enabled in this function